README.md

```diff
@@ -2143,102 +2171,57 @@ After enabling the client statistics, you can monitor your clients using Hazelca
-
-By default, `StreamHandler` outputs to the `sys.stderr`.
-
-> Note that `stream` argument is incompatible with the `filename` argument - if both are present, a `ValueError` is raised.
-
-Below is an example of this configuration.
-
-```python
-import sys
-
-logging.basicConfig(stream=sys.stdout)
-``` 
-
-#### Logging to File
-
-You can specify the `filename` and `filemode` attributes to log to a file. 
-
-When `filename` is set, a [FileHandler](https://docs.python.org/3/library/logging.handlers.html#logging.FileHandler), using the specified file name, will be created instead of a [StreamHandler](https://docs.python.org/3/library/logging.handlers.html#streamhandler). FileHandler sends the logging output to a disk file using the output functionality of the StreamHandler.
-
-If the `filename` is specified, you can also select the mode that the log file will be opened using the `filemode`. See the available [file modes](https://docs.python.org/3/library/functions.html#filemodes) in the Python documentation. 
-
-When `filemode` is not specified, `'a'` will be used by default.
-
-Below is an example of this configuration.
-
-```python
-logging.basicConfig(filename="/home/hazelcast/hz-py.log", filemode="w")
-```
-
-#### Specifying Logging Format
-
-Logging format can be set using the `format` and `datefmt` arguments. 
-
-Handlers will use the `format` string to specify the logging output. See the available format attributes in the [LogRecord attributes](https://docs.python.org/3/library/logging.html#logrecord-attributes) documentation. 
-
-Date and time format used in the format attributes such as `%(asctime)s` can be specified using `datefmt` argument. `datefmt` should be a legal format string for [time.strftime()](https://docs.python.org/3/library/time.html#time.strftime).
-
-Below is an example of this configuration and its output.
-
-```python
-logging.basicConfig(format='%(asctime)s%(msecs)03d [%(threadName)s][%(name)s] %(levelname)s: %(message)s', datefmt="%H:%M:%S,")
-``` 
-
-**Output**
-
-```
-08:48:12,438 [MainThread][LifecycleService] DEBUG: New Lifecycle state is STARTING
-08:48:12,439 [hazelcast-reactor][Reactor] DEBUG: Starting Reactor Thread
-```
+Let's go over the `LoggerConfig` options one by one.
 
 #### Setting Logging Level
 
-Although you can not change the logging levels used within the Hazelcast Python client, you can specify a logging level that will be used to threshold the logs that are at least as severe as your specified level using  the `level` argument.
+Although you can not change the logging levels used within the Hazelcast Python client, you can specify a logging level that will be used to threshold the logs that are at least as severe as your specified level using the `ClientConfig.logger_config.level`.
```

Comment:
No need to use "the" before `ClientConfig.logger_config.level`

(also you can use the simple present tense as a better structure)